include/ofi_util.h

```diff
@@ -247,6 +247,7 @@ struct util_ep {
 	struct util_cntr	*rem_rd_cntr; /* remote read   */
 	struct util_cntr	*rem_wr_cntr; /* remote write  */
 
+	enum fi_ep_type		type;
```

Comment:
Doesn't this change need to come before the previous commit?

Reply:
This is done in the first commit of this PR.